samples/Common.c

```diff
@@ -579,6 +584,27 @@ VOID sampleBandwidthEstimationHandler(UINT64 customData, DOUBLE maxiumBitrate)
     DLOGV("received bitrate suggestion: %f", maxiumBitrate);
 }
 
+VOID sampleSenderBandwidthEstimationHandler(UINT64 customData, UINT32 txBytes, UINT32 rxBytes, UINT32 txPacketsCnt, UINT32 rxPacketsCnt,
+                                            UINT64 duration)
+{
+    UNUSED_PARAM(customData);
+    UINT32 lostPacketsCnt = txPacketsCnt - rxPacketsCnt;
+    UINT32 percentLost = lostPacketsCnt * 100 / txPacketsCnt;
+    UINT32 bitrate = 1024;
+    if (percentLost < 2) {
+        // increase encoder bitrate by 2 percent
+        bitrate *= 1.02f;
+    } else if (percentLost > 5) {
+        // decrease encoder bitrate by packet loss percent
+        bitrate *= (1.0f - percentLost / 100.0f);
+    } else {
+        // keep bitrate the same
+    }
+
+    DLOGV("received sender bitrate estimation: suggested bitrate %u sent: %u bytes %u packets received: %u bytes %u packets in %lu msec, ", bitrate,
```

Comment:
Change to `DLOGS`?

Reply:
same as above - this entire message and even containing method can be removed
my goal was to explain what the method does to the user by proving a very verbose example